cmd/ctr/commands/containers/checkpoint.go

```diff
@@ -68,9 +69,6 @@ var checkpointCommand = cli.Command{
 		if context.Bool("rw") {
 			opts = append(opts, containerd.WithCheckpointRW)
 		}
-		if context.Bool("task") {
-			opts = append(opts, containerd.WithCheckpointTask)
-		}
```

Comment:
I do not believe this change is required. To restore correctly you should execute `ctr c restore --live ...`.

Reply:
I think the commands I listed above is misleading. Even we put `--live` in restore phase, the checkpoint doesn't have the layers expected.

Reply:
> I do not believe this change is required. To restore correctly you should execute `ctr c restore --live ...`.

you mean "--task", and not "--live" please?